src/test/java/com/bravo/user/util/TestUtil.java

```diff
@@ -0,0 +1,49 @@
+package com.bravo.user.util;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.User;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.model.dto.UserSaveDto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class TestUtil {
+
+    public static User getUserData() {
+
+        UserSaveDto userDto = new UserSaveDto();
+
+        userDto.setFirstName("Harrison");
+        userDto.setMiddleName("");
+        userDto.setLastName("Ford");
+        userDto.setPhoneNumber("");
+
+        return new User(userDto);
```

Comment:
Why did you create `UserSaveDto` instead of `User` directly? These variables can be `final` and if you use `@Builder` you can remove the variable altogether.

Reply:
Yes, I missed that one.